cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -197,6 +197,10 @@ spec:
                   type: integer
                 activeStandby:
                   type: boolean
+                activeStandbyCount:
+                  minimum: 1
```

Comment:
The activeStandbyCount is optional, right? If so I believe we need the min to be 0. If not, we will not be backward compatible.

Reply:
Yes. it is optional.